blimp-live-widgets/src/components/Weather/index.js

```diff
@@ -2,6 +2,11 @@ import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 
 import styles from './styles.css'
+import sunny from './assets/sunny.png'
```

Comment:
Is it possible to make these one line? like idk import {sunny, cloud, etc.} from ./asserts/ or something along those lines. If after like a 3 min search you can't easily find/it's not possible it's fine with me.

Reply:
No, according to this link: https://stackoverflow.com/questions/44607396/importing-multiple-files-in-react it would require a function, which isn't necessary

Reply:
I agree.